sessions/bringing-people-together/leveraging-r-based-ecosystem.md

```diff
@@ -7,9 +7,9 @@ talk_tags: [process, story, tidyverse, tool]
 session_slug: bringing-people-together
 # ---- Edit information below this line ----
 # The title of your talk
-talk_title: "Leveraging a R-based ecosystem in the US federal government"
+talk_title: "Building a Pit of Success for Your Organization: Embracing a R-based ecosystem in the US federal government"
 # A short version of the title, suitable for small displays
-talk_title_short: Leveraging a R-based ecosystem in the US federal government
+talk_title_short: Building a Pit of Success
```

Comment:
Would you consider

```suggestion
talk_title_short: Building a Pit of Success at USAID
```

or less specifically

```suggestion
talk_title_short: Building a Pit of Success for Your Organization
```

I think both are a little more clear if one were only able to read the short title.